stratum/hal/bin/barefoot/README.md

```diff
@@ -13,13 +13,12 @@ these targets.
 
 There are two different Stratum targets for Tofino:
 
-`stratum_bf` is based on the [PI](https://github.com/p4lang/PI) C++ API is the
-better tested and supported target. This target is still recommended for most
-users.
-
 `stratum_bfrt` is a newer target based on Barefoot's BfRt C++ API. This target
-is still experimental, but gives users access to additional Tofino features.
-The plan is to make this target the default when it has matured.
+is newer and less mature, but gives users access to additional Tofino features.
```

Comment:
I would remove "less mature."
Maybe say that this target is newer, actively used and tested, and gives users access...